sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -369,6 +370,48 @@ def has_ontime_pane(self):
     return False
 
 
+class AfterSynchronizedProcessingTime(TriggerFn):
+  """Fire exactly once after all worker clocks catch up the first element that arrives
+
+  AfterSynchronizedProcessingTime is experimental. No backwards compatibility guarantees.
+  """
+  def __repr__(self):
+    return 'AfterSynchronizedProcessingTime()'
+
+  def on_element(self, element, window, context):
+    # TODO: sort out whether this should be a NOOP or how to get the clock to be
+    # passed in
+    #context.set_timer(
+    #    '', TimeDomain.DEPENDENT_REAL_TIME, context.get_current_time())
```

Comment:
The `clock` is `None` in the context. Not sure where in the fn api runner that gets set. Not sure if I should just make this a noop or behave like Always trigger anyhow. But for my first draft I just went with what it should actually do, and that is crashy.